backend/controller/encryption/integration_test.go

```diff
@@ -31,6 +31,7 @@ func WithEncryption() in.Option {
 }
 
 func TestEncryptionForLogs(t *testing.T) {
+	t.Skip("This test needs the timeline service refactoring done")
```

Comment:
Encryption will be reimplemented, so you can probably jut delete this completely.